src/ontology/mondo-edit.obo

```diff
@@ -82347,12 +82347,12 @@ xref: MESH:D007119 {source="MONDO:equivalentTo", source="MONDO:ontobio"}
 xref: NCIT:C7528 {source="EFO:0000255", source="MONDO:equivalentTo", source="exact-label-match", source="DOID:0111147"}
 xref: ONCOTREE:AITL {source="MONDO:equivalentTo"}
 xref: Orphanet:86886 {source="MONDO:equivalentTo", source="DOID:0111147"}
+xref: PMID:22700722 {source="DOID:0111147"}
```

Comment:
We shouldn't have PMID xrefs for terms

Reply:
I moved this to a dbxref on the text def